image.hpp:30:10: fatal error: image_types.hpp: No such file or directory

`image.hpp` includes `image_types.hpp`, but the latter doesn't get installed:

```console
$ echo '#include <exiv2.hpp>' > test.cc
$ g++ -I/usr/local/include/exiv2 test.cc
In file included from /usr/local/include/exiv2/bmffimage.hpp:27,
                 from /usr/local/include/exiv2/exiv2.hpp:28,
                 from test.cc:1:
/usr/local/include/exiv2/image.hpp:30:10: fatal error: image_types.hpp: No such file or directory
   30 | #include "image_types.hpp"
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
```

Observed in the `main` branch (287744fa12142111f2ad2117ef455d5e5322e2c2).

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -59,6 +59,7 @@ set(PUBLIC_HEADERS
     ../include/exiv2/gifimage.hpp
     ../include/exiv2/http.hpp
     ../include/exiv2/image.hpp
+    ../include/exiv2/image_types.hpp
     ../include/exiv2/ini.hpp
     ../include/exiv2/iptc.hpp
     ../include/exiv2/jp2image.hpp

